scripts/importLanguages.py

```diff
@@ -1,84 +1,97 @@
 #! /usr/bin/env python3
-# Note: this wont work on python 2.7
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 # This script must be executed at the root of the repository.
 
+import argparse
 import xml.etree.ElementTree as ET
 import os
 import sys
 
-THRESHOLD = 0.70  # 70% Target Completeness for import
-FILES = []
-
-PROD = False
-PROD_LANGS = [ "en" ]
+prod_locales = ["en"]
+# Include only locales above this threshold (e.g. 70%)
+l10n_threshold = 0.70
 
-if len(sys.argv) > 1 and (sys.argv[1] == "-p" or sys.argv[1] == "--prod"):
-   PROD = True
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    '-p', '--prod', default=False, action="store_true", dest="isprod",
+    help='Build only for production locales.')
+args = parser.parse_args()
 
 # Step 1
-# Go through the i18n repo, check each xliff file and take
-# note which locale is >70% complete
-# Adds path of {.xliff / .ts } to FILES
+# Go through the i18n repo, check each XLIFF file and take
+# note which locale is complete above the minimum threshold.
+# Adds path of .xliff and .ts to l10n_files.
+l10n_files = []
 for locale in os.listdir('i18n'):
-    # Skip non folder
+    # Skip non folders
     if not os.path.isdir(os.path.join('i18n', locale)):
         continue
+
     # Skip hidden folders
     if locale.startswith('.'):
         continue
 
-    if PROD and not(locale in PROD_LANGS):
+    # Ignore locale if it's a production build and the locale should not be
+    # included
+    if args.isprod and locale not in prod_locales:
+        continue
+
+    xliff_path = os.path.join('i18n', locale, 'mozillavpn.xliff')
+
+    # If it's the source locale (en), ignore parsing for completeness and
+    # add it to the list.
+    if locale == 'en':
```

Comment:
The previous code was parsing `en` in this loop and excluding it because it's at 0%. Then adding it back in a a following loop, which seems unnecessary (it will always have zero translations).